src/pages/Home/index.js

```diff
@@ -36,6 +37,11 @@ const Home = () => {
     setLoading,
   });
 
+  const t = useTranslation();
+  const gameStatusMessage = t('home.gameStatusMessage', { gameStatus: gameStatus.toUpperCase() });
```

Comment:
I would use a memo here for all these kind of values across this PR to improve the performance a little bit

Reply:
If I understood correctly, done 👍🏻